Formula/qfs.rb

```diff
@@ -0,0 +1,101 @@
+class Qfs < Formula
+  desc "High-performance, fault-tolerant, distributed file system and its tools"
+  homepage "https://github.com/quantcast/qfs"
+  url "https://github.com/quantcast/qfs.git"
+  version "1.1.4-1"
+  head "https://github.com/quantcast/qfs.git"
+
+  bottle :unneeded
+
+  depends_on "boost"
+  depends_on "cmake" => :build
+  depends_on "maven" => :build
+  depends_on "openssl" => :build
+  depends_on "wget"
+
+  def install
+    ENV.java_cache
+
+    opoo "Building qfs. This may take some time..."
+    system "make", "build", "hadoop-jars", "BUILD_TYPE=release"
+
+    bin.install "build/release/bin/chunkscrubber"
```

Comment:
> I work at Quantcast and have had this discussion internally and I'm not sure if we are going to change it right now. We might in the future.

It'd be great if there was at least an upstream tracking issue for this. A `make install` is pretty standard practice to make life easier for packagers.